ckan/adx_config.ini

```diff
@@ -299,7 +298,7 @@ ckanext.saml2auth.logout_requests_signed = False
 # ckanext.saml2auth.key_file_path = /etc/ckan/saml.key
 # ckanext.saml2auth.cert_file_path = /etc/ckan/saml.crt
 # Uncomment the internal login option to enable log in in offline mode
-# ckanext.saml2auth.enable_ckan_internal_login = True
```

Comment:
That commented out option should stay True so it works according to the comment above.

Reply:
Apologies. Undone